docs/sections/user_guide/cli/mode_rocoto.rst

```diff
@@ -21,11 +21,16 @@ The ``uw`` mode for realizing and validating Rocoto XML documents
       validate
           Validate Rocoto XML
 
+
 .. _realize_rocoto_cli_examples:
 
 ``realize``
 -----------
 
+In ``uw`` terminology, to ``realize`` a configuration file is to transform it from its raw from into its final, usable state. In the case of ``uw rocoto``, that means transforming a structured YAML file into a :rocoto:`Rocoto XML<>` file. The structured YAML language required by UW closely follows the XML language defined by Rocoto.
+
+More information about the structured YAML file for Rocoto can be found :doc:`here<defining_a_workflow>`.
```

Comment:
This link does not seem to be rendering.

Reply:
I think I must respond to this one with the standard "it builds on my machine [mac]." 🙄 

I've checked out several things.
* Removed my build directory and ran `make html` from a clean state. 
* Checked that the RtD versions of packages were identical to those I have installed.
* Instead of running `make html` I ran the same command as RtD: `python -m sphinx -T -E -b html -d _build/doctrees -D language=en . $READTHEDOCS_OUTPUT/html`


I'm also seeing that RtD isn't providing any useful error with the latest push using `:any:` instead of `:doc:`. The log is here: https://readthedocs.org/api/v2/build/23050224.txt 

I also decided to test another machine, in case there's a weird Mac thing happening. I build fresh on Hera, too, for a linux build. While I didn't open it in a browser, I am still seeing that the HTML rendered appropriately there:

```<p>More information about the structured YAML file for Rocoto can be found <a class="reference internal" href="../rocoto_yaml.html#defining-a-workflow"><span class="std std-ref">here</span></a>.</p>```

On the RtD page, the line is:

```
<p>
    More information about the structured YAML file for Rocoto can be found <span class="xref std std-doc">here</span>
    .
</p>
```

Any tips/tricks on getting this to render on RtD?

Reply:
Interesting! I'll try it on my machine and see what happens.

Reply:
I'm looking at the RtD-rendered docs -- https://unified-workflow--376.org.readthedocs.build/en/376/sections/user_guide/cli/mode_rocoto.html -- and this looks fine to me now. What about you? Did it look broken at some point to you, too?

Reply:
You'll probably generally want to run `make docs` instead of `make html` now, too, so that the link checker also runs.

Reply:
I was running `make docs` early, but in the iterative process of troubleshooting, I stopped checking all the links...thanks for the reminder to return to it.

After loading the new page in an incognito window, and then refreshing in my regular tab, all is fine. Seems it was a chrome-y issue more than a platform one?